### PR TITLE
analytic content for TM ApexOne

### PR DESCRIPTION
**Pull request by spsocprime**
## Proposed Changes

  - analytic rules
  - hunting queries
  - workbook

### CODE REVIEW

**Review by shainw — COMMENTED** [review]
Approved from a detections perspective, however, I would suggest that for detections that have just a single condition, such as `| where EventMessage has "Suspicious File"` that you move these all into a single custom alert that changes the alert name and description based on the content of the query - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride

**Comment by vmanojreddy** [comment]
Workbooks Queries are failing. Please look into it.
image

**Comment by vmanojreddy** [comment]
@aprakash13 Please review analytic rule.

**Comment by spsocprime** [comment]
> Workbooks Queries are failing. Please look into it. image

If you do not have events in your environment the queries will fail because these are dynamic field names extracted from the event data

**Comment by aprakash13** [comment]
> custom alert

Shain has already reviewed this from a detection perspective. This looks good. There was a suggestion about using custom alert. However, that was a non-blocking comment. Thanks.

**Comment by NikTripathi** [comment]
@spsocprime Can you please share the updated sample data?

**Comment by spsocprime** [comment]
> @spsocprime Can you please share the updated sample data?

Updated sample data file

**Review by NikTripathi — CHANGES_REQUESTED** [review]

File: Sample Data/CEF/TrendMicro_ApexOne.json [hunk 1]
```diff
@@ -1,2837 +1,2620 @@
...
+        "DestinationMACAddress": "",
+        "DestinationNTDomain": "",
+        "DestinationProcessId": "",
+        "DestinationUserPrivileges": "",
+        "DestinationProcessName": "",
+        "DestinationPort": "",
+        "DestinationIP": "",
+        "DeviceTimeZone": "",
+        "DestinationUserID": "",
+        "DestinationUserName": "user@test.com",
```
[the remark this answers is not in the document]
Comment (spsocprime) [reply Sample Data/CEF/TrendMicro_ApexOne.json]: fixed

**Comment by NikTripathi** [comment]
Hi @spsocprime •	The validation error might be due to TMApexOneDvcAccessPermissionWasChanged.yaml file having relevant technique as T1401. This is not an Enterprise MITRE Techniques for Privilege Escalation and so the validator is failing. Only these are the valid options for PrivilegeEscalation currently.

Tactic.PrivilegeEscalation :  { "T1037", "T1053", "T1055", "T1068", "T1078", "T1134", "T1484", "T1543", "T1546", "T1547", "T1548", "T1574", "T0874", "T0890", "T1611" } 
Please try addressing these and see if the validation passes. Thanks.

**Comment by spsocprime** [comment]
> Hi @spsocprime • The validation error might be due to TMApexOneDvcAccessPermissionWasChanged.yaml file having relevant technique as T1401. This is not an Enterprise MITRE Techniques for Privilege Escalation and so the validator is failing. Only these are the valid options for PrivilegeEscalation currently.
> 
> Tactic.PrivilegeEscalation : { "T1037", "T1053", "T1055", "T1068", "T1078", "T1134", "T1484", "T1543", "T1546", "T1547", "T1548", "T1574", "T0874", "T0890", "T1611" } Please try addressing these and see if the validation passes. Thanks.

Fixed

**Pull request merged**